dotenv/Tiltfile

```
@@ -1,8 +1,46 @@
-def dotenv(fn='.env'):
-	f = read_file(fn)
-	lines = str(f).splitlines()
-	for line in lines:
-		v = line.split('=', 1)
-		if len(v) < 2:
-			continue # skip empty lines
-		os.putenv(v[0], v[1])
+def dotenv(fn=".env"):
+
+    # Check if file exists
+    if not os.path.exists(fn):
+        warn("Environment variables file not found: '%s'" % fn)
+        return
+
+    print("Loading environment variables from file: '%s'" % fn)
```

this seems like an odd thing to print?

---

The intention here was to provide visibility into which dotenv file is currently being loaded.

Our use case is to have a Git Mono Repo that contains Git Sub-Repos for each of the micro services. And we dynamically load the Titlfile from the sub-repos via the Tiltfile in the Mono Repo.

IMHO, this is still good to have. But if this seems to verbose then I can think of adding a `verbose` or `debug` parameter in the parent function that would provide this diagnostic information.

What are your thoughts?

```suggestion
    def dotenv(fn=".env", verbose=False):
    # ...
    if verbose:
        print("Loading environment variables from file: '%s'" % fn)
```